src/components/AddItem/index.jsx

```diff
@@ -1,39 +1,131 @@
+import React, { useState } from "react";
 import Header from "components/Header/index";
-import styles from "./style.module.css";
-import testimg from "../../assets/img/landing/Img_add.svg";
+import styles from "./addItem.module.css";
 
 function AddItems() {
+  const [productImg, setProductImg] = useState(null);
+  const [title, setTitle] = useState("");
+  const [info, setInfo] = useState("");
+  const [price, setPrice] = useState("");
+  const [tagInput, setTagInput] = useState("");
+  const [tags, setTags] = useState([]);
+
+  const handleImageUpload = (e) => {
+    if (e.target.files.length > 0) {
```

Comment:
혹시 files 가 없을 때를 대비해서 ? 를 붙여주는 걸 추천드려요!
```suggestion
    if (e.target?.files?.length ?? 0 > 0) {
```